commit 461db1515c66018266be9fee31b4f61e6cca60d4 (refs/changes/99/4399/1)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-06-21T16:54:51+01:00 (1 year, 4 months ago)
    
    MANTA-3781 manta-mackerel build asks for tar when it wants gtar

diff --git a/Makefile b/Makefile
index 00f283a..3ad09ea 100644
--- a/Makefile
+++ b/Makefile
@@ -119,7 +119,7 @@ publish: release
 
 .PHONY: assets
 assets: $(NODEUNIT)
-	tar -zcf $(ROOT)/assets/node_modules.tar node_modules build/node/bin/node
+	gtar -zcf $(ROOT)/assets/node_modules.tar node_modules build/node/bin/node
 
 
 include ./tools/mk/Makefile.deps
